testConstructorURLConnectionInput()
     tags: []
 uniqueId: [engine:junit-jupiter]/[class:org.apache.commons.io.input.XmlStreamReaderTest]/[method:testConstructorURLConnectionInput()]
   parent: [engine:junit-jupiter]/[class:org.apache.commons.io.input.XmlStreamReaderTest]
   source: MethodSource [className = 'org.apache.commons.io.input.XmlStreamReaderTest', methodName = 'testConstructorURLConnectionInput', methodParameterTypes = '']
   caught: java.net.UnknownHostException: www.apache.org
             	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
             	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
             	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:483)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1751)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1321)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1302)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
             	at org.apache.commons.io.input.XmlStreamReader.<init>(XmlStreamReader.java:674)
             	at org.apache.commons.io.input.XmlStreamReaderTest.testConstructorURLConnectionInput(XmlStreamReaderTest.java:285)
             	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
             	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
             	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
             Caused by: java.net.UnknownHostException: www.apache.org
             	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:569)
             	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:284)
             	at java.base/java.net.Socket.connect(Socket.java:659)
             	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:304)
             	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:181)
             	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:166)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:516)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:604)
             	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:206)
             	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:288)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:963)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:899)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1377)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1302)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:2866)
             	at java.base/java.net.URLConnection.getContentType(URLConnection.java:508)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getContentType(HttpsURLConnectionImpl.java:378)
             	at org.apache.commons.io.input.XmlStreamReader.<init>(XmlStreamReader.java:673)
             	... 4 more
 duration: 125 ms
   status: ✘ FAILED
testConstructorURLInput()
     tags: []
 uniqueId: [engine:junit-jupiter]/[class:org.apache.commons.io.input.XmlStreamReaderTest]/[method:testConstructorURLInput()]
   parent: [engine:junit-jupiter]/[class:org.apache.commons.io.input.XmlStreamReaderTest]
   source: MethodSource [className = 'org.apache.commons.io.input.XmlStreamReaderTest', methodName = 'testConstructorURLInput', methodParameterTypes = '']
   caught: java.net.UnknownHostException: www.apache.org
             	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
             	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
             	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:483)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1751)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1321)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1302)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
             	at org.apache.commons.io.input.XmlStreamReader.<init>(XmlStreamReader.java:674)
             	at org.apache.commons.io.input.XmlStreamReader.<init>(XmlStreamReader.java:648)
             	at org.apache.commons.io.input.XmlStreamReaderTest.testConstructorURLInput(XmlStreamReaderTest.java:297)
             	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
             	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
             	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
             Caused by: java.net.UnknownHostException: www.apache.org
             	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:569)
             	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:284)
             	at java.base/java.net.Socket.connect(Socket.java:659)
             	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:304)
             	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:181)
             	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:166)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:516)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:604)
             	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:206)
             	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:288)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:963)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:899)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1377)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1302)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:2866)
             	at java.base/java.net.URLConnection.getContentType(URLConnection.java:508)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getContentType(HttpsURLConnectionImpl.java:378)
             	at org.apache.commons.io.input.XmlStreamReader.<init>(XmlStreamReader.java:673)
             	... 5 more
 duration: 5 ms
   status: ✘ FAILED

--- a/src/test/java/org/apache/commons/io/input/XmlStreamReaderTest.java
+++ b/src/test/java/org/apache/commons/io/input/XmlStreamReaderTest.java
@@ -44,6 +44,7 @@ import java.util.Map;
 import org.apache.commons.io.CharsetsTest;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.io.function.IOFunction;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
@@ -280,6 +281,7 @@ class XmlStreamReaderTest {
         assertThrows(NullPointerException.class, () -> new XmlStreamReader((Path) null));
     }
 
+    @Disabled
     @Test
     protected void testConstructorURLConnectionInput() throws IOException {
         try (XmlStreamReader reader = new XmlStreamReader(new URL("https://www.apache.org/").openConnection(), UTF_8)) {
@@ -292,6 +294,7 @@ class XmlStreamReaderTest {
         assertThrows(NullPointerException.class, () -> new XmlStreamReader((URLConnection) null, US_ASCII));
     }
 
+    @Disabled
     @Test
     protected void testConstructorURLInput() throws IOException {
         try (XmlStreamReader reader = new XmlStreamReader(new URL("https://www.apache.org/"))) {
